src/app/features/landingpage/views/landing-page/landing-page.component.html

```diff
@@ -1,63 +1,76 @@
-      Das vierte dir entwandt!
-    </p>
-    <p><cite>Hans Andersen</cite></p>
-  </blockquote>
-
-  <p>
-    Höre nur, was die Jungen singen! sagten die kleinen Storchkinder. Sie sagen,
-    wir sollen gebraten und verbrannt werden!
-  </p>
-
-  <h3>Header Level 3</h3>
-  <ul>
-    <li>Das eine wird gepfählt</li>
-    <li>Das andere wird abgekehlt!</li>
-  </ul>
+<section class="bg-primary py-28 flex items-center justify-center">
+  <div class="rounded-lg bg-tint p-12 shadow-md">
+    <h1 class="text-xl pb-5">AHB Tabelle</h1>
+    <h2 class="text-[12px] border-b inline-block pb-[12px] mb-[20px] uppercase">
+      Anwendungshandbücher für Menschen
+    </h2>
+    <h3 class="font-bold pb-2">Finde dein AHB</h3>
+    <button
+      class="rounded-full bg-primary text-white px-5 py-2 shadow-md"
+      routerLink="/ahb"
+    >
+      Jetzt öffnen
+    </button>
+  </div>
 </section>
-<section
-  class="container py-10 px-10 mx-0 min-w-full flex flex-col items-center"
->
-  <h2 class="text-5xl mb-3 text-black">Finde dein AHB</h2>
-  <p class="text-black">lorem ipsum</p>
-  <a
-    routerLink="/ahb"
-    class="bg-primary text-white font-bold py-2 px-4 mt-3 rounded"
-  >
-    Jetzt oeffnen
-  </a>
+<section class="flex flex-col lg:flex-row bg-tint">
+  <div class="flex-1 px-10 pt-10">
+    <h4 class="font-bold text-sm mb-2">
+      Willkommen bei Ahbesser – Anwendungshandbücher für Menschen
+    </h4>
+    <p class="text-[12px]">
+      Bei uns steht der Mensch im Mittelpunkt. Unser Ziel ist es, die komplexen
+      und oft schwer verständlichen Anwendungshandbücher der deutschen
+      Energiewirtschaft benutzerfreundlich und zugänglich zu machen. Mit unserem
+      Service können Sie gezielt nach FormatVersionen und Prüfidentifikatoren
+      suchen und ein einzelnes Dokument nach spezifischen Details durchsuchen.
+      So sparen Sie wertvolle Zeit und Nerven, die Sie stattdessen in Ihr
+      Kerngeschäft investieren können.
+    </p>
+  </div>
+  <div class="flex-1 px-10 pt-10">
+    <h4 class="font-bold text-sm mb-2">
+      Effiziente Suche und einfache Navigation
+    </h4>
+    <p class="text-[12px]">
+      Wir wissen, dass das Durchforsten riesiger PDF-Dateien eine mühsame und
+      zeitraubende Aufgabe sein kann. Daher bieten wir Ihnen eine innovative
+      Plattform, die diesen Prozess erheblich vereinfacht. Unsere
+      benutzerfreundliche Suchfunktion ermöglicht es Ihnen, schnell und präzise
+      die Informationen zu finden, die Sie benötigen. Geben Sie einfach die
+      gewünschte FormatVersion oder den Prüfidentifikator ein, und schon
+      gelangen Sie zum relevanten Dokument – klar strukturiert und leicht
+      verständlich aufbereitet.
+    </p>
+  </div>
+  <div class="flex-1 px-10 pt-10">
+    <h4 class="font-bold text-sm mb-2">Ihre Vorteile auf einen Blick</h4>
+    <p class="text-[12px]">
+      Mit AHBesser wird die Arbeit mit Anwendungshandbüchern so effizient und
```

Comment:
```suggestion
      Mit AHB Tabellen wird die Arbeit mit Anwendungshandbüchern so effizient und
```
ich finde AHBesser auch cooler, aber den Name konnte ich nicht offiziell durchsetzen :(